### PR TITLE
PSK2 add home page

### PR DESCRIPTION
**Pull request by chuckh**
#This adds home page to PSK.  Changes select color to paper-indigo-800. Updates 404 to redirect to home page. Updates name to Polymer Starter Kit 2.

Screenshot of home page
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 <!doctype html>
 <html>
   <head>
-    <title>Polymer Starter Kit</title>
+    <title>Polymer Starter Kit 2</title>
```
Comment (robdodson) [inline index.html]: I think this will become PSK so not sure if we need to call it out as '2'

File: src/psk-app/psk-app.html [hunk 2]
```diff
@@ -193,12 +196,21 @@ <h4 class="title">Polymer Starter Kit - [[params.page]]</h4>
...
-          this.set('route.path', '/art');
+          this.set('route.path', '/');
         }
       },
 
       _drawerSelected: function() {
         if (!this.$.drawer.persistent) this.$.drawer.close();
+      },
+
+      _getPageName: function(pageName) {
```
Comment (robdodson) [inline src/psk-app/psk-app.html]: Thinking about this a bit. I think I'd prefer if we did not add features like this to the template for the sake of shipping the most minimal thing possible.

File: src/psk-app/psk-app.html [hunk 3]
```diff
@@ -173,12 +175,13 @@
 
           <!-- bottom toolbar -->
           <app-toolbar class="title-toolbar main-title-toolbar">
-            <h4 class="title">Polymer Starter Kit - [[params.page]]</h4>
+            <h4 class="title">Polymer Starter Kit 2 - [[_getPageName(params.page)]]</h4>
```
Comment (robdodson) [inline src/psk-app/psk-app.html]: Maybe stick with just Polymer Starter Kit

File: index.html (hunk 1 above)
Comment (chuckh) [inline index.html]: My thought was to to call it out for now while we have both. Then just before we merge it into master then remove it.

Comment (robdodson) [inline index.html]: I think we can go without it, I don't think there's much confusion and it's an additional thing we'll have to remember to take out late

Comment (abdonrd) [inline index.html]: I would not change. **Polymer Starter Kit** is okey for me.

**Comment by abdonrd** [comment]
Nice work @chuckh!

File: src/psk-app/psk-app.html (hunk 3 above)
Comment (chuckh) [inline src/psk-app/psk-app.html]: Done.

File: src/psk-app/psk-app.html (hunk 2 above)
Comment (chuckh) [inline src/psk-app/psk-app.html]: Done.

File: index.html (hunk 1 above)
Comment (chuckh) [inline index.html]: Done.

**Comment by chuckh** [comment]
@robdodson I made your suggested changes. Thanks @abdonrd!

File: .travis.yml [hunk 4]
```diff
@@ -1,14 +1,13 @@
 language: node_js
-sudo: false
+sudo: required
```
Comment (abdonrd) [inline .travis.yml]: I think it would be better to have it in a separate PR.

Comment (chuckh) [inline .travis.yml]: Just added separate PR #798 for this.

File: src/psk-app/psk-app.html [hunk 5]
```diff
@@ -193,13 +196,14 @@ <h4 class="title">Polymer Starter Kit - [[params.page]]</h4>
...
         if (!this.route.path) {
-          this.set('route.path', '/art');
+          this.set('route.path', '/');
         }
       },
 
       _drawerSelected: function() {
         if (!this.$.drawer.persistent) this.$.drawer.close();
-      }
+      },
```
Comment (abdonrd) [inline src/psk-app/psk-app.html]: `abdonrd-lint` modo on! 😅 

comma and line break. :)

**Comment by abdonrd** [comment]
Rebase and I think that we can merge! 🎉

File: src/psk-app/psk-app.html (hunk 5 above)
Comment (chuckh) [inline src/psk-app/psk-app.html]: Fixed. Rebasing next.

**Comment by abdonrd** [comment]
Rebase on `psk2` branch and I think that we can merge! 🎉 

screen shot 2016-04-19 at 01 14 23

**Comment by chuckh** [comment]
@abdonrd what git command do you recommend for rebase this with psk2?

**Comment by abdonrd** [comment]
@chuckh `git checkout psk2-add-home-page` and `git rebase psk2`. :)

**Comment by robdodson** [comment]
Yeah you pretty much never want to merge git branches into one another, instead you want to rebase which basically plays your changes on top of a previous branch.

For example (run these commands from your `psk2-add-home-page` branch.

```
git fetch origin (should fetch all of the latest branches)
git rebase origin/psk2 (replay your branches changes on top of the psk2 branch. This will make it seem like your changes come AFTER whatever is the current commit on psk2)
```

**Comment by robdodson** [comment]
I rolled these changes into 4ca12b2

**Comment by chuckh** [comment]
@robdodson thanks for the clarification and explanation.  I will be sure to use in the future!  😸
